Wikipedia/Code/NSDateFormatter+WMFExtensions.h

```diff
@@ -39,6 +39,6 @@
 
 + (instancetype)wmf_mediumDateFormatterWithoutTime;
 
-+ (instancetype)wmf_hyphenatedYearMonthDayFormatter;
++ (instancetype)wmf_englishHyphenatedYearMonthDayFormatter;
```

Comment:
it forces the strings to be english. the default is to use the current locale, which can result in output that doesn't use the [numeric system](https://en.wikipedia.org/wiki/Decimal) that's required in this case (e.g. Arabic instead of English)